openapi/paths/actor-builds/actor-builds@{buildId}.yaml

```diff
@@ -53,6 +53,12 @@ get:
   deprecated: false
   x-legacy-doc-urls:
     - https://docs.apify.com/api/v2#/reference/actor-builds/build-object/get-build
+  x-js-parent: BuildClient
+  x-js-name: get
+  x-js-doc-url: https://docs.apify.com/api/client/js/reference/class/BuildClient#get
+  x-py-parent: BuildClientAsync
```

Comment:
What was the decision behind using Async not sync clients?
I have no strong opinion on it, but I assume that Python developers can widely use the sync version.

Reply:
Interesting. It wasn't a decision. It was matched programmatically against the mapping table on a "first come, first served" basis. What is your suggestion?

<img width="554" alt="image" src="https://github.com/apify/openapi/assets/79609/53bfc037-3a34-49f5-80c9-5b4dac9964bb">

Reply:
It's caused by referencing the single API operation from multiple places in the Client docs. More of an issue of the Python one, rather than JS. I've highlighted the duplicate items in the [original mapping tables](https://docs.google.com/spreadsheets/d/1spP7jpuKDxiF0_lnHuCszHmETVCOq7fidApHPfpkpf0/edit?gid=107006499#gid=107006499).

Would you mind proposing how to remediate that @drobnikj, please?

Reply:
We discussed it with @fnesveda, and the async version looks fine for now. Let's ship it.